changelog/unreleased/issue-2468

```diff
@@ -0,0 +1,5 @@
+Issue: Compatibility with non global azure domain
```

Comment:
The prefix must be `Enhancement:`. I'd also reword the summary to `Enhancement: Add support for non-global Azure clouds`.